lib/browser/api/menu-utils.js

```diff
@@ -120,10 +120,10 @@ function sortItemsInGroup (group) {
   })
 
   const sortedNodes = sortTopologically(originalOrder, edges)
-  return sortedNodes.map(i => group[i])
+  return sortedNodes.map((i) => group[i])
```

Comment:
i'm a bit sad about losing no-paren functions

Reply:
It makes diffs cleaner in the long run though 😄